sema4ai/bin/create_env/conda.yaml

```diff
@@ -10,6 +10,7 @@ dependencies:
     - ruamel.yaml=0.18.6
     - msgspec=0.18.2
     - pip:
+          - robocorp==3.0.0
```

Comment:
I think that we shouldn't have a `robocorp` dep here... instead of using rcc to build the env to run `build_environment`, just use the standard python env from github actions and `pip install roborp` and then run directly with `python build_environment.py` should do the trick without having to change this (I'm worried that somehow we may end up adding that dep which isn't really needed for the extension to work).